packages/expo-av/src/Video.types.ts

```diff
@@ -175,6 +175,13 @@ export type VideoProps = {
    * A boolean which, if set to `true`, will display an image (whose source is set via the prop `posterSource`) while the video is loading.
    */
   usePoster?: boolean;
+  /**
+   * A React Component to display the poster image.
```

Comment:
```suggestion
   * A react-native `Image` like component to display the poster image.
```

since the component accepting a `source` prop like Image, maybe we should describe this a little clearer. it's not accepting all component types. at least from typescript, it will show an error in you passing a `View`